r-package/survlearners/R/surv_fl_grf.R

```diff
@@ -23,9 +23,11 @@
 #' censor.time <- (numeratorC/(4^2))^(1/2)
 #' Y <- pmin(failure.time, censor.time)
 #' D <- as.integer(failure.time <= censor.time)
+#' X.test <- matrix(rnorm(n * p), n, p)
```

Comment:
if you use n.test = something other than n then that is a nice simple extra test that everything works with test data different size than trainig data

Reply:
yes, done!